sass/compat/themes/cartsy/__imports.scss

```diff
@@ -0,0 +1,28 @@
+// ------------------------------------
+// DEFINE LOCAL VARIABLES
+// ------------------------------------
+$_body-page-selector:                  '.woocommerce-checkout' !default;
+$_body-theme-selector:                 '.theme-cartsy' !default;
+
+
+
+
+
+// ------------------------------------
+// DEFINE CSS VARIABLES
+// ------------------------------------
+:root {
+	--fluidcheckout--field--border-radius: #{ remsize( 0.6rem ) }; // Needed to use theme variable to match theme styles
+}
```

Comment:
- Maybe missing the variable for `field-height`? Unless the field height with this theme is exactly the same as the default value, it is likely that that variable would be needed here to ensure fields are all the same height including the `select2` fields and also to set the correct position for the inline field validation check icon.